README.md

```diff
@@ -235,26 +235,28 @@ suites:
           compliance: base/ssh
 ```
 
-### Use attributes with your inspec profiles
+### Use inputs with your inspec profiles
 
-To run a profile with attributes defined inline, you can adapt your `.kitchen.yml`:
+Note: InSpec Inputs were formerly known as InSpec Attributes. As they are not related to Chef Attributes, they have been renamed to reduce confusion.
```

Comment:
maybe explicitly show the yaml diff for anyone who already use this and need to change their `.kitchen.yml`?